src/app/modals/createwallet/createwallet.component.ts

```diff
@@ -1,4 +1,4 @@
-import { Component, Inject, forwardRef, ViewChild, ElementRef, ComponentRef } from '@angular/core';
+import { Component, Inject, forwardRef, ViewChild, ElementRef, ComponentRef, HostListener } from '@angular/core';
```

Comment:
Is hostlistener used?

Reply:
No, I thought to use it but forget to remove it. Will remove 

Thanks